modules/openxBidAdapter.js

```diff
@@ -13,6 +13,11 @@ export const spec = {
   code: BIDDER_CODE,
   supportedMediaTypes: SUPPORTED_AD_TYPES,
   isBidRequestValid: function(bid) {
+    if (bid.mediaType === VIDEO) {
```

Comment:
It's worth noting that there is another way to define a video bid now. I'm not going to hold up this PR though since it's been sitting for awhile. You can see the pattern here: https://github.com/prebid/Prebid.js/blob/master/modules/appnexusAstBidAdapter.js#L291

If you want, please open a new PR with this small change. Thanks